apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx

```diff
@@ -230,6 +232,12 @@ const NeverParticipatedInGovernanceVariant = () => {
   )
 }
 
+const useRerenderOnFocus = () => {
```

Comment:
This looks bad, I guess it's solving some bug that appears without it? Is there no other way?